src/hackflight.hpp

```diff
@@ -86,7 +90,50 @@ namespace hf {
             {
                 return fabs(_state.eulerAngles[axis]) < _ratePid->maxArmingAngle;
             }
-
+            
+            void checkBattery()
+            {
+              // Frequency management
+              static float lastTime = 0.0;
+              
+              // Battery management
+              static float lastTimeLowBattery = 0.0;
+              static float lastBatteryVoltage = 0.0;
+              static bool  isLastLowBattery = false;
+              
+              // Check battery with a freq. of 2Hz
+              if (_board->getTime() - lastTime > 0.5)
+              {
+                // Look is the battery is belowe the limit
+                if (_state.batteryVoltage < _BAT_MIN && _state.batteryVoltage > 2.5)
```

Comment:
Why is the part of the condition `_state.batteryVoltage > 2.5` required?

Reply:
To avoid trigger the _low_battery flag if connected through USB